### ISSUE TITLE
Statuses

### ISSUE THREAD

**Issue by lej333** [lej333/teambox#1]
Assign status to each e-mail communication such as:
- Open
- Pending response
- Closed

**Comment by Flare0** [comment]
Also each conversation have one status, if a conversation is closed and a new email is received should be re-opened again.

**Comment by Flare0** [comment]
I was thinking of a different approach but this can work well as well.
My idea was to have the inbox with all cases that is new or being worked on and that you have labels for the different "work" statuses and assignments. Then that you move it to another folder when it is closed.
And to only see the emails that you are working on you have buttons for show only "assigned to me", "unassigned" etc.

**Comment by Flare0** [comment]
Saw now that folders are not a thing in Gmail, apparently everything is just labels.

**Comment by lej333** [comment]
Yeah, you can just click on the labels when you want to see all assigned cases of an user. Yes I think it's indeed a good idea to work with labels for assignments, priorities and statuses instead of something customs. With labels we have more possibilities for filtering.

**Comment by lej333** [comment]
And the labels in GMail/GSuite can be grouped. So we can create three new groups for users, priorities and statuses. The customers can create new labels and put it under this three parents.
